jazkarta/shop/browser/templates/checkout_cart.pt

```diff
@@ -10,7 +10,7 @@
 <div class="portalMessage error"
      tal:condition="view/error|nothing">
   <strong>Error</strong>
-  ${view/error}
+  <span tal:content="string:${view/error}"></span>
```

Comment:
`<tal:s content="view/error" />`

(This avoids adding a span, and using `string:${}` isn't necessary since view/error is already a string.)